### PR TITLE
Extensions "What's New" page updates

### PR DESCRIPTION
**Pull request by dotproto**
This PR contains a number of updates from the past couple months. It also introduces some formatting changes to the extensions What's New page in an attempt to make it a little more information dense and to make the markup a little more semantic.

- [What's New](https://deploy-preview-1714--developer-chrome-com.netlify.app/docs/extensions/whatsnew) preview page
- [What's New](https://developer.chrome.com/docs/extensions/whatsnew) live page

### CODE REVIEW

**Review by nilanjansiromani — APPROVED** [review]

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:40]:
```suggestion
A number of APIs now support promises in Manifest V3.
```

Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:40]: Are these classes, methods, or properties? A heuristic I use is to only use 'API' for references to multiple item types. (Language should always be specific.) 

If any of these are methods they should have '()' after them.

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:59]: Same question here and throughout.

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:87]:
```suggestion
use `webRequest`, `webRequestBlocking`, and site-specific host permission to migrate to the
```

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:89]:
```suggestion
requiring the user to approve a new permissions grant.
```

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md:87]: The tech community likes to use 'leverage' as a synonym for 'use'. Unfortunately non-English speakers (and folks new to tech) won't find this definition in a dictionary and translations may or may not handle it correctly.

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:98]: >The `chrome.scripting` API's

Please be specific. What is an API called in the Extensions universe? Class? Object? Interface?

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:100]: Regarding 'main world', is there a more technical term for this? 'Main process?'

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:101]: >isolated world

Same question.

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md:98]: Please look at this throughout. This isn't just pickiness. You're missing an opportunity to provide information readers who are new to Extensions.

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:112]:
```suggestion
isolation](https://web.dev/cross-origin-isolation-guide/) to use features like
```

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:113]:
```suggestion
`SharedArrayBuffer`. Documentation on this feature is available for both [Manifest
```

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:115]:
```suggestion
V3](/docs/extensions/mv3/cross-origin-isolation/).
```

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:125]:
```suggestion
ruleset has been adjusted to give developers more flexibility. Previously
```

Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md:125]: Removal of 'implementation'. Again, this may seem minor. Distinguishing the implementation from the design is only relevant to audiences who deal with both. To a user of Extensions, it's not an implementation. It's just the way it works.

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:139]:
```suggestion
[`SharedArrayBuffer`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer). Opt in will be required starting in Chrome 95.
```

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:164]: >at some of the new capabilities it introduces

This could be tighter:

- at it's new capabilities.

or 

- at capabilities it introduces.

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:169]:
```suggestion
Chrome now supports JavaScript modules in service workers. To specify a module in
your manifest:
```

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:208]:
```suggestion
### Chrome 89: Customizable permissions for web accessible resources {: #customizable-permissions-for-mv3-web-accessible-resources }
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/whatsnew/index.md
Comment (rdcronin) [inline site/en/docs/extensions/whatsnew/index.md:25]:
```suggestion
[unregistering](/docs/extensions/reference/scripting/#method-unregisterContentScripts), and [getting
a list of](/docs/extensions/reference/scripting/#method-getRegisteredContentScripts) content scripts
```

File: site/en/docs/extensions/whatsnew/index.md
Comment (rdcronin) [inline site/en/docs/extensions/whatsnew/index.md:30]: We increased support here by some non-trivial factor.  Let's add some kind of qualifier - "Significantly expanded" for less fun, "Wildly expanded" (or similar) for more fun.

File: site/en/docs/extensions/whatsnew/index.md
Comment (rdcronin) [reply site/en/docs/extensions/whatsnew/index.md:40]: These are all "namespaces", but we generally refer to them as "APIs" - e.g. the tabs API, the history API, etc.  This is consistent with our language elsewhere.

File: site/en/docs/extensions/whatsnew/index.md
Comment (rdcronin) [inline site/en/docs/extensions/whatsnew/index.md:89]:
```suggestion
requiring the user to approve new permissions.
```

File: site/en/docs/extensions/whatsnew/index.md
Comment (rdcronin) [inline site/en/docs/extensions/whatsnew/index.md:91]: We're inconsistent in our date format here - some use 2021-10-27 and some use 2021.10.27.

I personally like the "look" of 2021.10.27 better, but that's purely aesthetic.

File: site/en/docs/extensions/whatsnew/index.md
Comment (rdcronin) [inline site/en/docs/extensions/whatsnew/index.md:111]:
```suggestion
Extensions and apps should now opt into [cross origin
```

We had to roll back the enforcement on this.  But we should still document / encourage developers to move.

Comment (rdcronin) [reply site/en/docs/extensions/whatsnew/index.md:111]: Actually, since we cover this below, maybe we just remove this...

**Review by rdcronin — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:30]: Happy to! I think in my original draft the intro for this section I said something like "This release contains more promise updates than any previous release." I think I'll change this header and tweak the intro paragraph to lean more into this.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:25]: Ack.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:28]: Ack.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:33]: Ack.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:40]: Ack.

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md:25]: Please keep the link on 'content scripts'. It's more descriptive than 'list'. Alternatively, you could link the whole phrase.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:86]: Thanks for the clarification and suggestion. I wanted to include this because it wasn't clear to me from the dNR docs why or when I would use this. I quite like this suggestion. I'd also to integrate this language (or something like it) into the dNR API docs in a separate PR.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:87]: Ack.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:89]: Ack.

Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:89]: Ack.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:91]: Oops, thanks for catching this. I was moving from the period separator to the dash because I'm not used to seeing dates in this format. That's personal preference, though.

At the moment I think it's better to avoid unnecessary churn and go back to periods.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:111]: Roger, removing for now.

More generally, I'd like to get better at noting when a feature becomes available in Canary, but that would have required more digging into individual issues and commit history than I was up for (or should do at all) last night.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:112]: Dropping this section per [@rdcronin's comment](https://github.com/GoogleChrome/developer.chrome.com/pull/1714/files#r746789544).

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:113]: Dropping this section per [@rdcronin's comment](https://github.com/GoogleChrome/developer.chrome.com/pull/1714/files#r746789544).

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:115]: Dropping this section per [@rdcronin's comment](https://github.com/GoogleChrome/developer.chrome.com/pull/1714/files#r746789544).

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:131]: Thanks for both the cleanup and typo fix. I realize now that while I wanted to explain the change, the original intent of this page was to provide developers with a running log of changes. Our plan was that for cases where additional information is necessary, this page would link out to other resources.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:125]: Dropping this language in favor of [@rdcronin's suggestion](https://github.com/GoogleChrome/developer.chrome.com/pull/1714/files#r746792080).

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:164]: Ack. Using "at it's new capabilities".

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:208]: Ack.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md]: Ack.

File: site/en/docs/extensions/whatsnew/index.md
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md]: I'm not sure if this is what @rdcronin had in mind, but as I think about these links a bit more I think I see a clear rationale for keeping "content scripts" outside of the link. This sentence provides a list of operations that are all related to content scripts, and each one links to a relevant method. By scoping the links to the operation and having "content scripts" outside of the link text, it's more obvious to the reader that each of the items in the list operates on the term that follows the list.

At the moment I have a slight to moderate preference for this structure:

> The [`chrome.scripting`] API now supports [registering], [updating], [unregistering], and [getting
a list of] content scripts at runtime.

Comment (rdcronin) [reply site/en/docs/extensions/whatsnew/index.md]: > This sentence provides a list of operations that are all related to content scripts, and each one links to a relevant method. By scoping the links to the operation and having "content scripts" outside of the link text, it's more obvious to the reader that each of the items in the list operates on the term that follows the list.

Yep, exactly this : )

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md:102]: You misunderstood the question. 'Main world' doesn't sound like any programmer speak I've ever heard. I wanted to make sure this is actually what it's called.

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md]: 'Namespaces' is something I never would have guessed. A writing best practice has always been to prefer the more specific term. In an electronic world, a more specific term results in fewer search results. It's an aid to finding information.

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:30]: I meant to catch this before it was merged. In numbers what do 'massively' and 'significantly more' actually mean? Did you double the number of promises? Higher? 

The suggestion was described as 'fun'. The problem is that one bit of fun in an otherwise stoic information dump has a danger of being read a hyperbole. We need to avoid that.

cc: @rdcronin

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/whatsnew/index.md
Comment (rdcronin) [reply site/en/docs/extensions/whatsnew/index.md:30]: More than doubled, I think.

How about:

Expanded promise support to 27 more APIs

File: site/en/docs/extensions/whatsnew/index.md
Comment (rdcronin) [reply site/en/docs/extensions/whatsnew/index.md:102]: "Main world" and "isolated world" are the canonical terms.

File: site/en/docs/extensions/whatsnew/index.md
Comment (rdcronin) [reply site/en/docs/extensions/whatsnew/index.md]: "APIs" is the canonical term for these.

File: site/en/docs/extensions/whatsnew/index.md
Comment (rdcronin) [inline site/en/docs/extensions/whatsnew/index.md:135]:
```suggestion
The [policy update blog post](/blog/policy-update-2sv/) published on 2021.06.29 has been updated to
```

**Review by rdcronin — APPROVED** [review]
LGTM, but please wait for Joe's sign off as well

File: site/en/docs/extensions/whatsnew/index.md
Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md:30]: Sounds good to me.
```suggestion
### Chrome 96: Expanded promise support to 27 more APIs {: #m96-promise-support }
```

**Review by jpmedley — APPROVED** [review]

**Pull request merged**
